### ISSUE TITLE
Npm package download

### ISSUE THREAD

**Issue by JFGHT** [ConsenSys-archive/drizzle-legacy#53]
I'm not sure but could there be an error on which files is npm downloading into a fresh node_modules?

> {
  "author": {
    "name": "Josh Quintal",
    "email": "josh@trufflesuite.com",
    "url": "http://truffleframework.com/drizzle"
  },
  "bundleDependencies": false,
  "deprecated": false,
  "description": "A reactive data store for web3 and smart contracts.",
  "keywords": [
    "ethereum",
    "redux",
    "redux-saga"
  ],
  "license": "ISC",
  "main": "drizzle.js",
  "name": "drizzle",
  "repository": {
    "type": "git",
    "url": "https://github.com/trufflesuite/drizzle"
  },
  "version": "1.1.5"
}

That is the package.json it's downloading, totally different from the original repo one plus a lot of files are missing.

**Comment by sliptype** [comment]
I think the idea is that the compiled version of drizzle on npm has all the dependencies in it already. The `package.json` you posted matches the one in `dist`. 

However, anything >1.1.0 does break for me:
```
./node_modules/drizzle/drizzle.js
Module not found: Can't resolve 'web3' in 'node_modules/drizzle'
```

I think some of the deps are missing. It looks like they are externals in [webpack.config.js](https://github.com/trufflesuite/drizzle/blob/master/webpack.config.js). Which means they are not being bundled, but are still expected to be in the environment.

This was changed in https://github.com/trufflesuite/drizzle/commit/2c45cc6f1ed8cca7c2270eb708850606e23f8bf7

**Comment by JFGHT** [comment]
@DiscRiskandBisque hey, any explanation for this?

**Comment by DiscRisknBisque** [comment]
This should now be fixed in v1.2.2. When the dependencies were unbundled, they weren't added to the production `pacakge.json` so were not installing unless otherwise present in the project already. Please re-open if I'm misinterpreting.
